[2023-06-17 21:02:08,176] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: task_flow_api._get_name 2023-06-16T00:00:00+00:00 [queued]>
[2023-06-17 21:02:08,192] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: task_flow_api._get_name 2023-06-16T00:00:00+00:00 [queued]>
[2023-06-17 21:02:08,194] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-06-17 21:02:08,195] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2023-06-17 21:02:08,196] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-06-17 21:02:08,205] {taskinstance.py:1063} INFO - Executing <Task(_PythonDecoratedOperator): _get_name> on 2023-06-16T00:00:00+00:00
[2023-06-17 21:02:08,209] {standard_task_runner.py:52} INFO - Started process 1931 to run task
[2023-06-17 21:02:08,214] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'task_flow_api', '_get_name', '2023-06-16T00:00:00+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpabc7ccyw', '--error-file', '/tmp/tmp2oyo9v2f']
[2023-06-17 21:02:08,218] {standard_task_runner.py:77} INFO - Job 9: Subtask _get_name
[2023-06-17 21:02:08,271] {logging_mixin.py:104} INFO - Running <TaskInstance: task_flow_api._get_name 2023-06-16T00:00:00+00:00 [running]> on host 7a0b2641fd5a
[2023-06-17 21:02:08,316] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Servin
AIRFLOW_CTX_DAG_ID=task_flow_api
AIRFLOW_CTX_TASK_ID=_get_name
AIRFLOW_CTX_EXECUTION_DATE=2023-06-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-16T00:00:00+00:00
[2023-06-17 21:02:08,341] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=task_flow_api, task_id=_get_name, execution_date=20230616T000000, start_date=20230617T210208, end_date=20230617T210208
[2023-06-17 21:02:08,371] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-17 21:02:08,386] {local_task_job.py:146} INFO - Task exited with return code 0
